src/compiler.cr

```diff
@@ -11,9 +11,9 @@ module Mint
     @static_components = {} of String => String
     @static_components_pool = NamePool(String, Nil).new
 
-    def initialize(@artifacts : TypeChecker::Artifacts, @optimize = false)
+    def initialize(@artifacts : TypeChecker::Artifacts, @optimize = false, css_prefix = "")
```

Comment:
```suggestion
    def initialize(@artifacts : TypeChecker::Artifacts, @optimize = false, css_prefix = nil)
```

Reply:
why is `nil` better than an empty string `""`?

Reply:
1. `instance_sizeof(String) # => 16` vs `sizeof(Nil) # => 0`, not too much but still ;)
2. semantically speaking, `nil` IMO better signifies the _lack of_ value then an empty string